src/content/docs/en/welcome.mdx

```diff
@@ -67,6 +58,26 @@ No, Ultramarine is not just Fedora with extra desktop environments. We aim to im
 
 Cutefish is no longer maintained upstream. We decided to stop supporting Cutefish due to this in Ultramarine 37.
 
+### What happened to Pantheon?
+
+We dropped our Pantheon Edition in Ultramarine 40. Downloads had been steadily declining for some time and Pantheon simply wouldn't work in 40, we decided to drop it for these reasons.
+
+### Can you add a (insert desktop here) edition?
+
+Put simply, maybe. We require any new desktop being added to Ultramarine to fit a small set of requirements:
+
+- Be actively maintained (Release in the last year, plans for another release, reccent commits)
+- Be capable of displaying libhelium apps (The installer and out-of-box experience are both Helium apps)
+- A working Wayland session **OR** a reasonable plan to implement one
+
+Ideally, any new edition will also have the following:
+
+- Accessibility toolings
+- A native app store with support for distribution upgrades
+- Support for the accent colours and colour schemes setting portals
+
+It should be noted a new Ultramarine desktop edition is rare. If is a new desktop in the Linux space, that doesn't mean Ultramarine will immediately, or ever, adopt it, as new desktop editions require a lot of planning, development, testing, etc.

```

Comment:
remove this

Reply:
the last two requirements aren't changing, we should rewrite line 79 there to make it a little less stringent